opts.go

```diff
@@ -271,3 +310,19 @@ func loadFromConfDir(c *libcni, max int) error {
 	c.networks = append(c.networks, networks...)
 	return nil
 }
+
+func isValidSize(filepath string, maxSize int64) (bool, error) {
+	file, err := os.Open(filepath)
+
+	if err != nil {
+		return false, err
```

Comment:
not there is .. fine should fail to exist in a different way

Reply:
by is fine, I mean since the file does not exist it is not larger than max and should be ignored here .. just return true...